<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8QIQ87H" failures="3" tests="3" name="Test" time="9.022" errors="0" timestamp="2023-06-01T15:24:11 PKT">
  <testcase classname="UserManagement.UserManagemantTest" name="Test01" time="2.659">
    <failure type="java.lang.AssertionError" message="expected [200] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at UserManagement.UserManagemantTest.Test01(UserManagemantTest.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- Test01 -->
  <testcase classname="UserManagement.UserManagemantTest" name="Test02" time="5.018">
    <failure type="java.lang.AssertionError" message="expected [200] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at UserManagement.UserManagemantTest.Test02(UserManagemantTest.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- Test02 -->
  <testcase classname="UserManagement.UserManagemantTest" name="Test03" time="1.235">
    <failure type="java.lang.AssertionError" message="expected [401] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [401] but found [500]
at UserManagement.UserManagemantTest.Test03(UserManagemantTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- Test03 -->
</testsuite> <!-- Test -->
